### DISCUSSION TITLE
Gauge initialization fails in .NET 6.0

### DISCUSSION THREAD

**Discussion by ayankumaar** [getgauge/gauge#2234] [General]
I have recently installed .NET 6.0 in my system. I have completely uninstalled .NET 5.0 and other versions from my system. In my gauge project written using C# I am targeting .NET 6.0. However, in the lsp.log file I can see the following error:

**13-04-2022 17:14:27.259 [dotnet] [ERROR] It was not possible to find any compatible framework version
13-04-2022 17:14:27.260 [dotnet] [ERROR] The framework 'Microsoft.AspNetCore.App', version '5.0.0' (x64) was not found.
13-04-2022 17:14:27.262 [dotnet] [ERROR] - The following frameworks were found:
13-04-2022 17:14:27.262 [dotnet] [ERROR] 6.0.3 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
13-04-2022 17:14:27.262 [dotnet] [ERROR] You can resolve the problem by installing the specified framework and/or SDK.
13-04-2022 17:14:27.262 [dotnet] [ERROR] The specified framework can be found at:
13-04-2022 17:14:27.262 [dotnet] [ERROR] - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=5.0.0&arch=x64&rid=win10-x64**

I am using the following:

Gauge version: 1.4.3
Commit Hash: f98dd40

Plugins
-------
dotnet (0.4.1)
html-report (4.1.4)
screenshot (0.0.1)

.NET SDK: 6.0.201
VS Code: 1.66.1
gauge VS Code extension: 0.1.0
OS: Windows 10

I was just checking if the latest version of gauge supports .NET 6.0 ? If not then is there any update when it is going to support it ?

**Comment by zabil** [comment]
This is discussed here https://github.com/getgauge/gauge-dotnet/issues/178

You should be able to build from source to get .NET 6 support.
